optimum/bettertransformer/models/__init__.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from . import bart, bert, distilbert
+
+
+FAST_LAYERS_MAPPING_DICT = {
```

Comment:
Not sure about the "fast" name, could be confusing, maybe `...BetterTransformer` could be fine? As it seems it's the communication name from PyTorch: https://pytorch.org/blog/PyTorch-1.13-release/#stable-bettertransformer-api

Reply:
I was just afraid the naming would be too long in this case :D

Reply:
Fixed in 1b87882dc27b9ce9bc6bba7665493c24d9693dae